templates/core/terraform/appgateway/appgateway.tf

```diff
@@ -107,7 +107,7 @@ resource "azurerm_application_gateway" "agw" {
     protocol                                  = "Https"
     path                                      = "/api/health"
     timeout                                   = "30"
-    unhealthy_threshold                       = "3"
+    unhealthy_threshold                       = "10"
```

Comment:
I'm curious why we've increased this threshold from 3 to 10.

Previously we'd wait 45 seconds (15 x 3) before declaring the API unhealthy
Now we'll wait 150 seconds.

Have we picked this number based upon something related to the addition of Service Bus to the health endpoint?

Reply:
You are right. My bad. For some reason I only looked at the 3 and didn't do the 15*3 calculation. 45 seconds should be enough even with the Service Bus addition. Returned it to the original value